src/components/Comments.jsx

```diff
@@ -1,18 +1,34 @@
 import { useState } from "react";
 import { useContext } from "react";
 import { UserContext } from "../contexts/UserContext";
-import { postCommentByArticleId } from "../utils/apicalls";
+import { postCommentByArticleId, deleteCommentById } from "../utils/apicalls";
 
 const Comments = ({ article_id, comments, setComments }) => {
   const { user, setUser } = useContext(UserContext);
 
   const [posting, setPosting] = useState("");
   const [postDisabled, setPostDisabled] = useState(false);
+  const [deleteDisabled, setDeleteDisabled] = useState(false);
```

Comment:
It appears you aren't actually using this state value to change any of the information your component is rendering. 
You are choosing to display the delete button based on whether or not the username matches the logged in user. 
This (and any references to this state) can be removed without affecting the functionality